desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -153,29 +176,30 @@ def _make_result_table(result):
 
     table.append(pivot_row)
 
-  return tabulate(table, tablefmt="plain")
+  return tabulate(table, headers=['Columns({})'.format(idx+1), '', ''], tablefmt="simple")
```

Comment:
Column count to give a column estimate @romainr 
<img width="613" alt="Screenshot 2021-03-19 at 8 48 13 PM" src="https://user-images.githubusercontent.com/42064744/111802815-6e5e3e80-88f4-11eb-8947-9e957090a8a1.png">